.pre-commit-config.yaml

```diff
@@ -17,18 +17,20 @@ repos:
     hooks:
       - id: ruff
         args: ["--fix"]
+        exclude: ^(src/diracx/client/)
 
   - repo: https://github.com/psf/black
     rev: 23.9.1
     hooks:
       - id: black
+        exclude: ^(src/diracx/client/)
```

Comment:
Why? The client generation isn't being modifed and relies on this to format it. Also there is manually written code in `client/` (mainly the patch files) which should definitely be linted and formatted.

Reply:
From previous discussions the suggestion was to avoid linting this dir. Anyway, fine for me to remove this line.